docs/how-to-start.md

```diff
@@ -67,10 +67,12 @@ services:
       - /usr/lib:/host/usr/lib:ro
       - /sys/:/host/sys/:ro
       - /dev:/host/dev:ro
+      - type: tmpfs
+        target: /module
     depends_on:
       - grpc-server-debug
   grpc-server-debug:
-    image: stackrox/grpc-server:latest
+    image: quay.io/rhacs-eng/grpc-server:latest
```

Comment:
This is a private image unfortunately, so people outside the rhacs-eng org won't be able to use it. Maybe we can add a comment pointing at the main repo make target for people to know how to build it from source? In my experience that is usually fast enough.

Reply:
Good point, I added a clarification below.